spec/features/allocations_spec.rb

```diff
@@ -5,13 +5,16 @@
 
   scenario "Accredited provider views PE allocations page" do
     given_accredited_body_exists
+    given_training_provider_with_pe_fee_founded_course_exists
     given_i_am_signed_in_as_an_admin
```

Comment:
Might be worth a comment here as this isn't really the feature. It will be `given_i_am_signed_in_as_a_user_from_the_accredited_body` or something like that. Can we note that this is a placeholder too as you have elsewhere?